frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTable.component.tsx

```diff
@@ -200,6 +201,21 @@ export const TicketsTable = () => {
 		TicketsCardActionsDispatchers.setReadOnly(readOnly);
 	}, [readOnly]);
 
+	useEffect(() => {
+		ticketsFilteredByTemplate.forEach(({ modelId, modelName, ...ticket }) => {
+			if (!ticket.properties[BaseProperties.STATUS]) {
+				TicketsActionsDispatchers.updateTicket(
+					teamspace, 
+					project,
+					modelId,
+					ticket._id,
+					getTicketWithStatus(ticket, selectedTemplate),
```

Comment:
This is necessary because getticketWithStatus is done in the fetch instead of the selector.